src/components/navbar/navbar.component.tsx

```diff
@@ -94,6 +104,24 @@ const Navbar: React.FC<NavbarProps> = ({
                 >
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
+                  aria-label="Toggle App Menu"
+                  onClick={handleSwitcherClick}
+                >
+                  <Icon />
+                </HeaderGlobalAction>
+                <HeaderPanel
+                  aria-label="Toggle App Menu Panel"
+                  style={{
+                    display: switcherToggler ? "block" : "none",
+                    height: "12rem",
+                    overFlow: "overflow"
+                  }}
+                  expanded
+                >
+                  <ExtensionSlot extensionSlotName="toggle-app-menu-slot"></ExtensionSlot>
```

Comment:
Just `app-menu-slot` (the toggle part is really not interesting, nor may it be accurate as it reflects the current state of the implementation / behavior).

Also this is an empty element; just self-close it.